fluidsynth/src/sfloader/fluid_defsfont.h

```diff
@@ -435,7 +435,7 @@ struct _fluid_defpreset_t
 {
   fluid_defpreset_t* next;
   fluid_defsfont_t* sfont;                  /* the soundfont this preset belongs to */
-  char name[21];                        /* the name of the preset */
+  char name[22];                        /* the name of the preset */
```

Comment:
The SF2 spec says a preset name has 20 characters without having counted the terminal '\0' (see `The PHDR Sub-chunk`, page 21). So changing this to 22 actually is a mistake IMO. Where got this format overflow warning triggered?